app/api/dds/v1/app_api.rb

```diff
@@ -60,7 +59,7 @@ class AppAPI < Grape::API
               Sneakers::CONFIG[:retry_error_exchange],
               ApplicationJob.distributor_exchange_name
             ].each do |expected_exchange|
-              unless Sneakers::CONFIG[:connection].exchange_exists? expected_exchange
+              unless ApplicationJob.conn.exchange_exists? expected_exchange
```

Comment:
:wrist_slap: ApplicationJob.conn is a private method...

Also, if the problem is that the connection isn't started, there should be a test that demonstrates that.  :P